homeassistant/components/camera/netatmo.py

```diff
@@ -43,9 +40,10 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         return None
 
     for camera_name in data.get_camera_names():
-        if config[CONF_CAMERAS] != []:
-            if camera_name not in config[CONF_CAMERAS]:
-                continue
+        if CONF_CAMERAS in config:
```

Comment:
Can you simplify this by moving it outside of the loop (it won't change during the loop) and also convert to be 1 if statement.